andela/bp-esa-frontend
# #164238429 Fix Sign Out button

#### What does this PR do?
Fixes sign out button by closing it when out of focus and aligns the dropdown icon on the filter buttons to the text

#### Description of Task to be completed?
- align dropdown icon in filter button with the text
- close sign out button when out of focus

#### How should this be manually tested?
1. Open the application on your browser
2. Sign in
3. On the header, click on the dropdown button on the far right
4. A sign out button should appear
5. Click away from the button
6. The sign out button should close/ disappear
7. Check that the dropdown icons on the filter buttons is well aligned with the button text
8. Check the filter and search buttons to ensure that they are interactive
9. Run `npm test` to ensure that the tests pass

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
[#164238429](https://www.pivotaltracker.com/n/projects/2197045/stories/164238429)
